toolchain/lex/tokenized_buffer.h

```diff
@@ -169,26 +169,15 @@ class TokenizedBuffer : public Printable<TokenizedBuffer> {
   // Returns the comment's full text range.
   auto GetCommentText(CommentIndex comment_index) const -> llvm::StringRef;
 
-  // Prints a description of the tokenized stream to the provided `raw_ostream`.
+  // Returns tokens as YAML. This prints the tracked token information on a
+  // single line. We use the single-line format so that output is compact, and
```

Comment:
This makes it sound like all the tokens go on the same line

```suggestion
  // Returns tokens as YAML. This prints the tracked token information on a
  // single line for each token. We use the single-line format so that output is compact, and
```